Sources/StreamChatUI/CommonViews/ContainerView.swift

```diff
@@ -28,15 +28,16 @@ extension UILayoutPriority {
 }
 
 extension CGFloat {
-    static let auto: CGFloat = .infinity
+    public static let auto: CGFloat = .infinity
 }
 
 public class ContainerView: UIView {
-    convenience init(
+
+    public convenience init(
         axis: NSLayoutConstraint.Axis = .horizontal,
         alignment: Alignment = .fill,
         spacing: CGFloat = .auto,
-        views: [UIView] = []
+        views: [UIView]
```

Comment:
This doesn't look correct :)

Reply:
Haha now I see what was in the conflict while rebasing :D Fixed now...